src/service/templates/models/rethinkdb/templates/service.model.js

```diff
@@ -0,0 +1,27 @@
+const path = require('path');
+const rethink = require('rethinkdbdash');
+
+module.exports = function (options) {
+
+  // Create a rethinkdb instance
+  const r = rethink({
+    db: options.db // @slajax todo: add servers to config
+  });
+
+  // <!--- This code can be removed!
+  // All it does is set up your DB table.
+  r.dbList().contains(options.db) // create db if not exists
```

Comment:
I think we should move this into a standalone utility inside the `feathers-rethinkdb` repo, so you can just do something like this.  It's really ugly code (+1 for hiding it behind a module), but handy for portability (between dbs) and cheap to keep in the app initialization.

```js
const createTable = require('feathers-rethinkdb/create-table');

createTable('tableName', r).then(...)
```
@slajax @ekryski @daffl thoughts?